api/proxy.go

```diff
@@ -86,6 +109,51 @@ func buildProxiedURL(uri, kubernetesID, resourceName, namespace string) string {
 	return fmt.Sprintf("/apis/everest.percona.com/v1alpha1/namespaces/%s%s%s", namespace, uri, resourceName)
 }
 
+func everestModifier(resp *http.Response) error {
```

Comment:
I think we should rename this function to something more meaningful, it should allude to the modification it does which is this case is to change the status message if we face an error.
Something like
```suggestion
func mapStatusMessageCRDsOnError(resp *http.Response) error {
```